src/python/zquantum/core/bitstring_distribution_test.py

```diff
@@ -153,10 +172,40 @@ def test_uses_epsilon_instead_of_zero_in_target_distribution():
     with mock.patch("core.bitstring_distribution.math.log", log_spy):
         target_distr = BitstringDistribution({"000": 0.5, "111": 0.4, "010": 0.0})
         measured_dist = BitstringDistribution({"000": 0.1, "111": 0.9, "010": 0.0})
+        distance_measure_params = {"epsilon": 0.01}
+        compute_clipped_negative_log_likelihood(
+            target_distr, measured_dist, distance_measure_params
+        )
 
-        compute_clipped_negative_log_likelihood(target_distr, measured_dist, epsilon=0.01)
+        log_spy.assert_has_calls(
+            [mock.call(0.1), mock.call(0.9), mock.call(0.01)], any_order=True
+        )
+
+
+@pytest.mark.parametrize(
+    "measured_dist,distance_measure_params,expected_mmd",
```

Comment:
I don't see a test testing if `sigma` defaults to `1.0` as it is.
I think there should either be a test for that or the default value should be removed.

Reply:
I agree. I don't know how to make it here though, since `sigma` is a local variable within the `compute_mmd` function (same for `epsilon` in `compute_clipped_negative_log_likelihood`). Any suggestions? 😅

Reply:
I think you can calculate it twice - once with `sigma` explicitly set to `1.0` and the other time without passing any arguments. 
Then you can compare if the results from both cases match each other.

Reply:
Done it [here](https://github.com/zapatacomputing/z-quantum-core/pull/63/commits/1c3e5168aefda3cdee23839089c41addcfd83c93)

Reply:
Addressed [here](https://github.com/zapatacomputing/z-quantum-core/pull/63/commits/3955916b079b25bef7f8294b03752cb70ae21852)